modules/genai-ecosystem/pages/langchain4j.adoc

```diff
@@ -40,6 +40,56 @@ An example is avalable at: https://github.com/langchain4j/langchain4j-examples/t
 include::https://github.com/langchain4j/langchain4j-examples/raw/main/neo4j-example/src/main/java/Neo4jEmbeddingStoreExample.java[]
 ----
 
+=== Usage Examples
+
```

Comment:
Scrivi anche qualcosa tipo:

To create a `Neo4jEmbeddingStore`, we can write:
Neo4jEmbeddingStore embeddingStore = Neo4jEmbeddingStore.builder().\<builderParameters>.build();

where \<builderParameters> can have ....

e fai riferimento ai parametri che si trovano in Neo4jEmbeddingStore.java alla riga 74.


Poi scrivi anche che in alternativa a `driver` possiamo usare `withBasicAuth` (come sta documentato sulla riga 64)